sys/fscontrol.c

```diff
@@ -614,14 +624,18 @@ NTSTATUS DokanMountVolume(__in PDEVICE_OBJECT DiskDevice, __in PIRP Irp) {
     RtlCopyMemory(dokanControl.UNCName, dcb->UNCName->Buffer,
                   dcb->UNCName->Length);
   }
-  mountEntry = FindMountEntry(dcb->Global, &dokanControl);
-  if (mountEntry != NULL) {
+  mountEntry = FindMountEntry(dcb->Global, &dokanControl, TRUE);
+  if (mountEntry != NULL && &mountEntry->MountControl) {
```

Comment:
But MountControl is not a pointer only mountEntry is, it can be zero-filled but its address will never be NULL.